ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/rtb/AppLovinRtbAdapter.java

```diff
@@ -0,0 +1,124 @@
+package com.applovin.mediation.rtb;
+
+import android.app.Application;
+import android.text.TextUtils;
+import android.util.Log;
+
+import com.applovin.mediation.AppLovinUtils;
+import com.applovin.sdk.AppLovinSdk;
+import com.google.ads.mediation.sample.adapter.BuildConfig;
+import com.google.android.gms.ads.AdSize;
+import com.google.android.gms.ads.mediation.rtb.AdRenderingCallback;
+import com.google.android.gms.ads.mediation.rtb.BannerAd;
+import com.google.android.gms.ads.mediation.rtb.BannerEventListener;
+import com.google.android.gms.ads.mediation.rtb.InterstitialAd;
+import com.google.android.gms.ads.mediation.rtb.InterstitialEventListener;
+import com.google.android.gms.ads.mediation.rtb.RewardedAd;
+import com.google.android.gms.ads.mediation.rtb.RewardedEventListener;
+import com.google.android.gms.ads.mediation.rtb.RtbAdConfiguration;
+import com.google.android.gms.ads.mediation.rtb.RtbAdapter;
+import com.google.android.gms.ads.mediation.rtb.RtbConfiguration;
+import com.google.android.gms.ads.mediation.rtb.RtbSignalData;
+import com.google.android.gms.ads.mediation.rtb.SignalCallbacks;
+import com.google.android.gms.ads.mediation.rtb.VersionInfo;
+
+import java.util.List;
+
+/**
+ * Created by Thomas So on July 17 2018
+ */
+public class AppLovinRtbAdapter
+        extends RtbAdapter
+{
+    private static final String TAG = "AppLovinRtbAdapter";
+
+    // TODO: Why is this not in the base adapter?!
+    // @Override
+    // public void setUp()
```

Comment:
I don't see this method in `RtbAdapter` in the latest version of the GMA SDK. I'm guessing that this method may have been dropped in favor of `initialize()`

Reply:
:/ inconsistent with iOS - we are going to figure out what's up.